composer.json

```diff
@@ -9,7 +9,7 @@
         "aws/aws-sdk-php": "~3.0",
         "dfurnes/environmentalist": "0.0.2",
         "doctrine/dbal": "^2.6",
-        "dosomething/gateway": "3.0.0",
+        "dosomething/gateway": "^1.17",
```

Comment:
You should be able to bump to `^2.2.1` if you want! And then we can always bump to `^3.0` when we get this application upgraded to Laravel 6.x sometime later this quarter.

Reply:
I tried this but the test import form dies when trying to request the logged-in user:

```
(1/1) ErrorExceptionimplode(): Invalid arguments passed
--
in Northstar.php line 59
at HandleExceptions->handleError(2, 'implode(): Invalid arguments passed', '/home/vagrant/Code/chompy/vendor/dosomething/gateway/src/Northstar.php', 59, array('id' => 'id', 'fields' => '5dd8d819fdce276a557e5895'))
at implode(',', '5dd8d819fdce276a557e5895')in Northstar.php line 59
```

Seems like i'd need to make some tweaks in the codebase for this version, going to go back to the good ol' days for now.

Reply:
Yep, it [had a slight API change](https://github.com/DoSomething/gateway/releases/tag/v2.0.0). Definitely nothing we need to tackle now.